### PR TITLE
Provides some Fixes and Improvements for Tagliatelle and Tycho.

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/resources/default/templates/system/info.html.pasta [hunk 1]
```diff
@@ -21,6 +22,7 @@
         </t:pageHeader>
     </i:block>
 
+    <i:local name="webContext" value="CallContext.getCurrent().get(WebContext.class)"/>
```
Comment (mkeckmkeck) [inline src/main/resources/default/templates/system/info.html.pasta:25]: Why is this required?

**Review by mkeckmkeck — APPROVED** [review]

File: src/main/resources/default/taglib/t/codeEditor.html.pasta [hunk 2]
```diff
@@ -13,11 +13,13 @@
 
 <script type="text/javascript">
     sirius.ready(function() {
-        let editor = ace.edit('@id');
+        let node = document.getElementById('@id');
```
Comment (sabieber) [inline src/main/resources/default/taglib/t/codeEditor.html.pasta:16]: Optional: Rename node to _node

**Review by sabieber — APPROVED** [review]

**Review by sabieber — APPROVED** [review]

**Review by sabieber — APPROVED** [review]
PR Improvements made:
- Added labels
- Improved title
